<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GUIAdaptador</class>
 <widget class="QMainWindow" name="GUIAdaptador">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>300</width>
    <height>500</height>
   </size>
  </property>
  <property name="focusPolicy">
   <enum>Qt::TabFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Adaptador R232 - Socket</string>
  </property>
  <property name="locale">
   <locale language="Spanish" country="Colombia"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>281</width>
      <height>241</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Velocidad : </string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Tipo de Flujo : </string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Paridad : </string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Bits : </string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="comboVelocidad">
       <item>
        <property name="text">
         <string notr="true">300</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">600</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">1200</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">2400</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">4800</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">9600</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">19200</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">38400</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">57600</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">115200</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="comboTipo">
       <item>
        <property name="text">
         <string>Ninguno</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">RTS/CTS</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">Xon/Xoff</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QComboBox" name="comboParidad">
       <item>
        <property name="text">
         <string>Ninguna</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Par</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Impar</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QComboBox" name="comboData">
       <item>
        <property name="text">
         <string notr="true">5</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">6</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">7</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">8</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QComboBox" name="comboStop">
       <item>
        <property name="text">
         <string notr="true">1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">2</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Puerto : </string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="comboPuerto">
       <item>
        <property name="text">
         <string notr="true">/dev/ttyUSB0</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">/dev/ttyS0</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">/dev/ttyS1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">/dev/ttyS2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">/dev/ttyS3</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Bit de Parada : </string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>280</y>
      <width>62</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>SOCKET</string>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>281</width>
      <height>93</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="spacing">
      <number>6</number>
     </property>
     <item row="2" column="0">
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>Puerto</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="lineEditPuerto">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEditServidor">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>Servidor : </string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>400</y>
      <width>281</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="0">
      <widget class="QPushButton" name="pushButtonIniciar">
       <property name="text">
        <string>Iniciar</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="pushButtonDetener">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Detener</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="pushButtonSalir">
       <property name="text">
        <string>Salir</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>10</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>RS232</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>281</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>390</y>
      <width>280</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>300</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArchivo">
    <property name="title">
     <string>Archivo</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionCerrar"/>
   </widget>
   <widget class="QMenu" name="menuEdici_n">
    <property name="title">
     <string>Edición</string>
    </property>
    <widget class="QMenu" name="menuIdioma">
     <property name="title">
      <string>Idioma</string>
     </property>
     <addaction name="actionEspanol"/>
     <addaction name="actionIngles"/>
     <addaction name="actionPortugues"/>
    </widget>
    <addaction name="separator"/>
    <addaction name="menuIdioma"/>
   </widget>
   <widget class="QMenu" name="menuAyuda">
    <property name="title">
     <string>Ayuda</string>
    </property>
    <addaction name="actionAyuda"/>
    <addaction name="separator"/>
    <addaction name="actionAcerca_de"/>
    <addaction name="actionAcerca_de_Qt"/>
   </widget>
   <addaction name="menuArchivo"/>
   <addaction name="menuEdici_n"/>
   <addaction name="menuAyuda"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAbrir">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Abrir</string>
   </property>
  </action>
  <action name="actionGuardar">
   <property name="text">
    <string>Guardar</string>
   </property>
  </action>
  <action name="actionCerrar">
   <property name="text">
    <string>Salir</string>
   </property>
  </action>
  <action name="actionAyuda">
   <property name="text">
    <string>Ayuda</string>
   </property>
  </action>
  <action name="actionAcerca_de">
   <property name="text">
    <string>Acerca de</string>
   </property>
  </action>
  <action name="actionEspanol">
   <property name="text">
    <string>Español</string>
   </property>
  </action>
  <action name="actionIngles">
   <property name="text">
    <string>Ingles</string>
   </property>
  </action>
  <action name="actionPortugues">
   <property name="text">
    <string>Portugues</string>
   </property>
  </action>
  <action name="actionAcerca_de_Qt">
   <property name="text">
    <string>Acerca de Qt</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
